content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -249,13 +249,14 @@ well as lower-trust users.The following listed controls should be enforced/disal
 		<tr>
 			<td>Seccomp</td>
 			<td>
-				The 'runtime/default' seccomp profile must be required, or allow specific additional profiles.<br>
+				The RuntimeDefault seccomp profile must be required, or allow specific additional profiles..<br>
 				<br><b>Restricted Fields:</b><br>
-				metadata.annotations['seccomp.security.alpha.kubernetes.io/pod']<br>
-				metadata.annotations['container.seccomp.security.alpha.kubernetes.io/*']<br>
+				spec.securityContext.seccompProfile.type<br>
+				spec.containers[*].securityContext.seccompProfile.type<br>
+				spec.initContainers[*].securityContext.seccompProfile.type<br>
 				<br><b>Allowed Values:</b><br>
 				'runtime/default'<br>
-				undefined (container annotation)<br>
+				undefined / nil<br>
```

Comment:
Considering `.type`, would the allowed values not be: `RuntimeDefault`, `Unconfined` and `Localhost`? If we refer to a level up (a.k.a `.seccompProfile`) then we can probably just leave `undefined/nil`.

Reply:
Hmm good point. Yeah I think what we are trying to say here (though I feel this document is a bit confusing) is that you can skip adding a `securityContext.seccomp` section to your containers. To communicate this do you just think we should remove `.type` off of the `initContainers` and `containers` lines?

Reply:
yep, I think if we just remove `.type` it becomes clear enough. :-1: